config/commands/goal.yaml

```diff
@@ -17,10 +17,17 @@ command:
     -
       name: create
       description: Create a new goal.
-      usage: create [<title>]
+      usage: create [options] [<title>]
```

Comment:
All commands and subcommands implicitly have a `--help` option, so, yes, there is at least one option. And ideally, I'd like us to always illustrate that.

The usage message is a kind of simplified grammar. Hopefully the examples clarify.